modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -190,34 +190,38 @@ Using random access, you can change a record's payload at any given index positi
 
 ==== Considerations for Defining a Block Size
 
-On a traditional on-line processing model, each request is usually mapped to a worker thread. Regardless of the processing type (either synchronous, asynchronous, one-way, request-response or even if the requests are temporarily buffered before being processed), servers usually end up in a 1:1 relationship between a request and a running thread. +
-When it comes to a batch job, all records are first stored in a persistent queue before the Process phase begins, so that the traditional threading model wouldn't apply.
+In a traditional online processing model, each request is usually mapped to a worker thread. Regardless of the processing type (either synchronous, asynchronous, one-way, request-response or even if the requests are temporarily buffered before being processed), servers usually end up in a 1:1 relationship between a request and a running thread.
```

Comment:
this shouldn't be in this page. Block size has absolutely nothing to do with filtering and aggregation. It doesn't even have to do with batch steps. It's a performance concern